CMakeLists.txt

```diff
@@ -3,25 +3,22 @@ project (OpenMEEG VERSION 2.4.9999 LANGUAGES C CXX)
 
 set(CMAKE_CXX_STANDARD 11)  # use c++11
 
-set(OpenMEEG_HEADER_INSTALLDIR include) # XXX I don't know why this should be here
-
 # Modify where to find the modules
 list(APPEND CMAKE_MODULE_PATH
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake/utils
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FortranCInterface
-    ${CMAKE_MODULE_PATH}
 )
 
 ################
 # OpenMEEG Configuration and needed things
 ################
 include(GenerateExportHeader)
-include(cmake/CompilerOptions.cmake)
-include(cmake/options.cmake)
-include(cmake/thirdParties.cmake)
-include(cmake/install.cmake)  # install setup (do it before adding targets)
-include(cmake/Uninstall.cmake)  # install setup (do it before adding targets)
+include(CompilerOptions)
```

Comment:
I am -1 in using only the file name. Explicit is better than implicit, and if your cmake paths are messed up (like having duplicated files in `cmake_old` for historical purposes) then it's not always clear.